default/scripting/buildings/shipyards/ORBITAL_DRYDOCK.focs.txt

```diff
@@ -15,6 +15,24 @@ BuildingType
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
+        // With no pop, no repair or sitrep messages
+        EffectsGroup
+            scope = And [
+                Ship
+                InSystem id = Source.SystemID
+                OwnedBy empire = Source.Owner
+                Structure high = LocalCandidate.MaxStructure - 0.001
+            ]
+            activation = And [
+                Source
+                ContainedBy And [
+                    Planet
+                    Population high = 0
+                ]
+            ]
+            stackinggroup = "SHIP_REPAIR"
+            priority = 75
```

Comment:
Hm, guess we'll have to live with that solution then, but I still foresee potential issues...